src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -240,6 +234,10 @@ class ProjectTeams extends AsyncView {
     return (
       <div>
```

Comment:
Can this become the `<Panel>`?

Reply:
no because the `SettingsPageHeader` shouldn't be inside of the `Panel`